docs/_interface/Private-Network.md

```diff
@@ -153,146 +119,353 @@ adapt to the amount of mining resources you deploy on the chain.
 
-To create a member node running on the same machine as the bootstrap node, choose a
-separate data directory (example: `data-2`) and listening port (example: `30305`):
+For example, using data directory (example: `data2`) and listening port (example: `30305`):
 
 ```shell
-geth --datadir data-2 --networkid 15 --port 30305 --bootnodes <bootstrap-node-record>
+geth --datadir data2 --networkid 12345 --port 30305 --bootnodes <bootstrap-node-record>
 ```
 
-With the member node running, you can check whether it is connected to the bootstrap node
-or any other node in your network by attaching a console and running `admin.peers`. It may
-take up to a few seconds for the nodes to get connected.
+With the member node running, it is possible to check that it is connected to the bootstrap node or any other node in the network by attaching a console and running `admin.peers`. It may take up to a few seconds for the nodes to get connected.
 
 ```shell
-geth attach data-2/geth.ipc --exec admin.peers
+geth attach data2/geth.ipc --exec admin.peers
 ```
 
-### Clique: Running A Signer
+### Running A Signer (Clique)
 
-To set up Geth for signing blocks in proof-of-authority mode, a signer account must be
-available. The account must be unlocked to mine blocks. The following command will prompt
-for the account password, then start signing blocks:
+To set up Geth for signing blocks in Clique, a signer account must be available. The account must already be available as a keyfile in the keystore. To use it for signing blocks, it must be unlocked. The following command, for address `0x7df9a875a174b3bc565e6424a0050ebc1b2d1d82` will prompt for the account password, then start signing blocks:
 
 ```shell
 geth <other-flags> --unlock 0x7df9a875a174b3bc565e6424a0050ebc1b2d1d82 --mine
 ```
 
-You can further configure mining by changing the default gas limit blocks converge to
-(with `--miner.gastarget`) and the price transactions are accepted at (with `--miner.gasprice`).
+Mining can be further configured by changing the default gas limit blocks converge to (with `--miner.gastarget`) and the price transactions are accepted at (with `--miner.gasprice`).
 
-### Ethash: Running A Miner
+### Running A Miner (Ethash)
 
-For proof-of-work in a simple private network, a single CPU miner instance is enough to
-create a stable stream of blocks at regular intervals. To start a Geth instance for
-mining, run it with all the usual flags and add the following to configure mining:
+For PoW in a simple private network, a single CPU miner instance is enough to create a stable stream of blocks at regular intervals. To start a Geth instance for mining, it can be run with all the usual flags plus the following to configure mining:
 
 ```shell
 geth <other-flags> --mine --miner.threads=1 --miner.etherbase=0x0000000000000000000000000000000000000000
```

Comment:
Can you please use a different account here to address this issue https://github.com/ethereum/go-ethereum/pull/25106? maybe one of the alloc accounts

Reply:
sure - done.